Deploy/stacks/dynamic/examples/datasets/inputs/config/mapbox-footprint.sql

```diff
@@ -0,0 +1,31 @@
+WITH "uuid_table" AS (
+    SELECT
+        "strval" AS "uuid",
+        "cityobject_id"
+    FROM
+        "cityobject_genericattrib"
+    WHERE
+        "attrname" = 'uuid'
+),
+"iri_table" AS (
+    SELECT
+        "urival" AS "iri",
+        "cityobject_id"
+    FROM
+        "cityobject_genericattrib"
+    WHERE
+        "attrname" = 'iri'
+)
+SELECT
+    "building"."id" AS "building_id",
+    COALESCE("measured_height", 100.0) AS "building_height",
```

Comment:
building_height is non-standard? this is the string that needs to be configured in the visualisation file

Reply:
Correct, measured_height would be CityGML standard (although we calculate it instead of measuring it). Since users need to provide their own SQL file for geoserver publication anyway, I don't think this is a problem - they could rename it to anything they want.